ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -55,10 +56,30 @@ class BottomSheetDemoController: UIViewController {
     private let personaListView: PersonaListView = {
         let personaListView = PersonaListView()
         personaListView.personaList = samplePersonas
+        personaListView.backgroundColor = Colors.NavigationBar.background
         personaListView.translatesAutoresizingMaskIntoConstraints = false
         return personaListView
     }()
 
+    private let headerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = Colors.gray100
+        view.heightAnchor.constraint(equalToConstant: headerHeight).isActive = true
+
+        let label = Label()
+        label.text = "Header view"
+        label.font = .systemFont(ofSize: 24)
```

Comment:
please remove hardcoded font size because our demo app still goes through accessibility reviews. use one of the text ramp. i think label by default is .body